package.json

```diff
@@ -186,7 +186,7 @@
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-storybook": "^0.6.13",
-    "glob": "^10.4.5",
+    "glob": "^11.0.0",
```

Comment:
**Note**: I had to update something to bust the npm dependency cache (this seemed like the least risky)